vscode-extension/editor/src/VSCodeEditor.tsx

```diff
@@ -101,6 +103,11 @@ export default function VSCodeEditor() {
         // /get endpoint so we get the latest content from the server
         return;
       }
+      case "set_theme": {
+        const theme = message.theme;
+        setThemeMode(theme);
```

Comment:
I think we should save this in webview state similar to how we save the server url (see line 100). Otherwise when the webview is refreshed on tab switch it will have the wrong theme.

Reply:
Ah, thanks, nice catch